macros/src/lib.rs

```diff
@@ -458,3 +459,141 @@ impl Parse for OptionSetter {
         Ok(Self { attrs, name, type_ })
     }
 }
+
+#[import_tokens_attr]
+#[proc_macro_attribute]
+pub fn option_setters_2(
```

Comment:
Is `option_setters` still being used somewhere or can it be deleted after making this `option_setters_2`?

Reply:
Once all of the actions have been moved over I'll delete `option_setters` and rename `option_setters_2` to replace it.